processor/src/trace/mod.rs

```diff
@@ -109,6 +109,15 @@ impl ExecutionTrace {
         result
     }
 
+    /// Returns helper registers state at the specified `clk` of the VM
+    pub fn get_helper_state(&self, clk: usize) -> [Felt; NUM_HASHER_COLUMNS - 2] {
```

Comment:
A few small nits:
* Instead of doing `NUM_HASHER_COLUMNS - 2`, we should probably create a constant in `core::decoder` for the number of helper registers. Maybe name it `NUM_USER_OP_HELPERS`?
* Similarly `DECODER_TRACE_OFFSET + 10` we should have a constant in `core::decoder` for this.
* I'd prefer to change the method name to `get_user_op_helpers_at`.